Release 1.12.0(develop) (#580)
SwedbankPay/developer.swedbankpay.com: resources/release-notes.md

body="The version numbers used in headers on this page refers to the version of
this very documentation, not to a version of any APIs described by it." %}

> ## July 2020

Reviewer: Codacy found an issue: [[no-empty-sections] Remove empty section: "July 2020"](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

## Version 1.12.0

*   Added information on 3-D Secure 2 for [Checkout][checkout-3ds2] and 

>     Card Payments[card-3ds2]. 

Reviewer 2: ```suggestion
    [Card Payments][card-3ds2]. 
```

*   Updated reconciliation files in [Settlement &
    Reconciliation][settlement-reconcilitation].

> *   Added testcard for Forbrugsforeningen in [Test data][test-data].

Reviewer 2: ```suggestion
*   Added test card for Forbrugsforeningen in [Test data][test-data].
```

> *   Documented `paid`/`failed`/`aborted` operations for all Payments. 

Reviewer 2: ```suggestion
*   Documented `paid`, `failed` and `aborted` across all resources.
```

*   Added additional information on `payerReference`, `generateRecurrenceToken`,
    `paymentToken`, `generatePaymentToken`, `recurrenceToken` and

>     `instrument`for `PaymentOrder` operations in [Checkout][checkout].

Reviewer 2: ```suggestion
    `instrument` for the `paymentorder` resource in [Checkout][checkout].
```

> *   Corrected and updated `view`- operations for all Payment instruments. 

Reviewer 2: ```suggestion
*   Corrected and updated `view-` operations for all resources. 
```

> *   Updated [Seamless View Events][seamless-view-events-card] for all Payments.

Reviewer 2: ```suggestion
*   Updated [Seamless View Events][seamless-view-events-card] for all resources.
```

> *   Added `nonPaymentToken` and `externalNonPaymentToken` description in

Reviewer 2: ```suggestion
*   Described `nonPaymentToken` and `externalNonPaymentToken` in
```

    Seamless View][card-seamless-view] sequence diagrams.
*   Added [Abort][swish-abort] in [Swish Payments][swish].

## 29 May 2020

### Version 1.11.0

*   Added a new [Trustly Payments][trustly-payments] section.
*   Added information about [Storing URIs][storing-uri] in the [Technical
    Information][home-technical-information] section.

## 22 May 2020

### Version 1.10.1

*   Updated [Swish Redirect][swish-redirect] requests.
*   Added `orderReference` information in Payment Order Callback in [Checkout Other features][checkout-callback].

## 14 May 2020

### Version 1.10.0

*   Added a new [Gift Cards][gift-cards] section with descriptions of our Gift Cards API.
*   Added a new section [Co-badge Card Choice for Dankort][co-badge-card] in
    [Card Payments][card-payment-url].

## 07 May 2020

### Version 1.9.2

*   Corrected table in [payments page][payments]

> ## 07 May 2020

Reviewer: Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (87:1)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

[swish-other-features]: /payments/swish/other-features
[swish-redirect]: /payments/swish/redirect
[swish-seamless-view]: /payments/swish/seamless-view

> [seamless-view-events-card]: payments/card/other-features#seamless-view-events

Reviewer 2: Codacy is saying this link doesn't work.

Author: corrected!
